### PR TITLE
Feature/point

### PR DESCRIPTION
**Pull request by limtaewon**

### CODE REVIEW

File: src/main/java/com/nhnacademy/bookpubshop/member/repository/impl/MemberRepositoryImpl.java [hunk 1]
```diff
@@ -34,18 +34,20 @@
...
+    QAuthority authority = QAuthority.authority;
+
+
+
     public MemberRepositoryImpl() {
         super(Member.class);
     }
 
 
     @Override
```
Comment (unhas01) [inline src/main/java/com/nhnacademy/bookpubshop/member/repository/impl/MemberRepositoryImpl.java:50]: Java Doc

File: src/main/java/com/nhnacademy/bookpubshop/member/repository/impl/MemberRepositoryImpl.java [hunk 2]
```diff
@@ -60,9 +62,6 @@ public Optional<MemberDetailResponseDto> findByMemberDetails(Long memberNo) {
...
```
Comment (unhas01) [inline src/main/java/com/nhnacademy/bookpubshop/member/repository/impl/MemberRepositoryImpl.java:67]: Java Doc

File: src/main/java/com/nhnacademy/bookpubshop/member/repository/impl/MemberRepositoryImpl.java [hunk 3]
```diff
@@ -183,9 +173,6 @@ public LoginMemberResponseDto findByMemberLoginInfo(String id) {
...
```
Comment (unhas01) [inline src/main/java/com/nhnacademy/bookpubshop/member/repository/impl/MemberRepositoryImpl.java:181]: Java Doc

File: src/main/java/com/nhnacademy/bookpubshop/point/repository/PointHistoryRepositoryCustom.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+/**
+ * 포인트 내역 레포지토리 커스텀.
+ *
+ * @author : 임태원
+ * @since : 1.0
+ **/
+@NoRepositoryBean
+public interface PointHistoryRepositoryCustom {
+    Page<GetPointResponseDto> getPointHistory(
```
Comment (unhas01) [inline src/main/java/com/nhnacademy/bookpubshop/point/repository/PointHistoryRepositoryCustom.java:24]: Java Doc

**Review by unhas01 — APPROVED** [review]
👍

**Review by hobby3530 — APPROVED** [review]
수고하셨습니다!

**Pull request merged**